config/opal_config_asm.m4

```diff
@@ -27,13 +27,13 @@ dnl
 dnl This is a C test to see if 128-bit __atomic_compare_exchange_n()
 dnl actually works (e.g., it compiles and links successfully on
 dnl ARM64+clang, but returns incorrect answers as of August 2018).
-AC_DEFUN([OPAL_ATOMIC_COMPARE_EXCHANGE_N_TEST_SOURCE],[
+AC_DEFUN([OPAL_ATOMIC_COMPARE_EXCHANGE_N_TEST_SOURCE],[[
 #include <stdint.h>
 #include <stdbool.h>
 #include <stdlib.h>
 
 typedef union {
-    uint64_t fake@<:@2@:>@
+    uint64_t fake@<:@2@:>@;
```

Comment:
Oops!  Good catch; thanks. 

Note that with the use of the quadrigraphs, the double quoting `[[]]` is not necessary.

Reply:
Without the double `[[]]` only the first few lines of the C code are in the configure test so all tests fails because the code is incomplete. I didn't try to understand why, I went for the simplest protection mechanism, the double `[[]]`.

Reply:
Huh; weird.  I played with the quoting quite a bit -- for me, using Autoconf 2.69, the double quoting was not necessary.  Shrug.  It's not harmful; might as well leave it.